src/WebEid.AspNetCore.Example/Certificates/CertificateLoader.cs

```diff
@@ -0,0 +1,42 @@
+﻿namespace WebEid.AspNetCore.Example.Certificates
+{
+	using System.Collections.Generic;
+	using System.IO;
+	using System.Linq;
+	using System.Security.Cryptography.X509Certificates;
+
+	internal static class CertificateLoader
```

Comment:
Duplicate code in web-eid-authtoken-validation-dotnet. Can/should this be exposed in web-eid-authtoken-validation-dotnet?

Reply:
In web-eid-authtoken-validation-dotnet certificates are loaded from resources, but here we read files